helm/notifications-push/app-configs/notifications-push_eks_delivery.yaml

```
@@ -4,15 +4,18 @@ service:
   name: notifications-push
 env:
   TOPIC: "PublicationNotificationEvents"
-  METADATA_TOPIC: "PostConceptAnnotations"
   NOTIFICATIONS_RESOURCE: "content"
-  CONTENT_URI_WHITELIST: "^http://(methode|wordpress-article|content|upp)(-collection|-content-placeholder|-notifications-creator)?(-mapper|-unfolder|)(-pr|-iw)?(-uk-.*)?\\.svc\\.ft\\.com(:\\d{2,5})?/(content|complementarycontent)/[\\w-]+.*$"
-  CONTENT_TYPE_WHITELIST: "application/vnd.ft-upp-article-internal+json,application/vnd.ft-upp-content-package+json,application/vnd.ft-upp-audio+json,application/vnd.ft-upp-live-blog-post-internal+json,application/vnd.ft-upp-live-blog-package-internal+json"
-  WHITELISTED_METADATA_ORIGIN_SYSTEM_HEADERS: "http://cmdb.ft.com/systems/pac, http://cmdb.ft.com/systems/methode-web-pub, http://cmdb.ft.com/systems/next-video-editor"
+  API_URL_RESOURCE: "content"
+  CONTENT_URI_ALLOWLIST: "^http://(content|upp)(-collection|-content-placeholder|-notifications-creator)?(-mapper|-unfolder|)(-pr|-iw)?(-uk-.*)?\\.svc\\.ft\\.com(:\\d{2,5})?/(content|complementarycontent)/[\\w-]+.*$"
```

```suggestion
  CONTENT_URI_ALLOWLIST: "^http://(content|upp)(-collection|-content-placeholder|-notifications-creator)?(-mapper|-unfolder|)\\.svc\\.ft\\.com(:\\d{2,5})?/(content|complementarycontent)/[\\w-]+.*$"
```

Can be simplified even further but I'd not take the chances with that many changes in a single PR.

---

At a second look, this would always expect a `(-mapper|-unfolder|)`. Is it even working properly prior to this PR?

---

The `|)` probably is what allows that group to be empty 😄 . I can change it to `(-mapper|-unfolder)?` so it's cleaner, it shouldn't affect how it works.

I was also debating about `(-pr|-iw)?(-uk-.*)?` but I'm not sure if it will affect something. As Vankata mentioned it might be worth to collect all the URIs that we can publish content with so we can update these URIs

---

Ah, you are probably right about the `|)`... 😄 

`(-pr|-iw)?(-uk-.*)?` was part of the content types published by Methode in the part.